GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -738,136 +742,185 @@ - (void)testNotHandleWrongPath {
+        NSArray<NSString *> *strings = [bodyString componentsSeparatedByString:@"="];
+        XCTAssertEqualObjects(strings[1], kAccessToken);
+
+        NSData *data = [[NSData alloc] init];
+        response(data, nil);
+        [expectation1 fulfill];
+     };
+  [_httpFetcher setTestBlock:testBlock];
+  
+  XCTestExpectation *expectation =
+      [self expectationWithDescription:@"Callback called with nil error"];
+  [_signIn disconnectWithCompletion:^(NSError * _Nullable error) {
+    XCTAssertNil(error);
+    [expectation fulfill];
+  }];
+  [self waitForExpectationsWithTimeout:1 handler:nil];
+  XCTAssertNil([_keychainHandler loadAuthState]);
+}
+
 // Verifies disconnect calls callback with no errors if access token is present.
-- (void)testDisconnect_accessToken {
+- (void)testDisconnect_accessTokenIsPresent {
   [_keychainHandler saveAuthState:_authState];
-  [[[_authState expect] andReturn:_tokenResponse] lastTokenResponse];
-  [[[_tokenResponse expect] andReturn:kAccessToken] accessToken];
-  [[[_authorization expect] andReturn:_fetcherService] fetcherService];
+  OCMStub([_authState lastTokenResponse]).andReturn(_tokenResponse);
+  OCMStub([_tokenResponse accessToken]).andReturn(kAccessToken);
+  
+  GIDHTTPFetcherTestBlock testBlock =
+      ^(NSURLRequest *request, GIDHTTPFetcherFakeResponseHandlerBlock responseHandler) {
+        NSData *data = [[NSData alloc] init];
+        responseHandler(data, nil);
+     };
+  [_httpFetcher setTestBlock:testBlock];
+  
   XCTestExpectation *expectation =
       [self expectationWithDescription:@"Callback called with nil error"];
   [_signIn disconnectWithCompletion:^(NSError * _Nullable error) {
-    if (error == nil) {
-      [expectation fulfill];
-    }
+    XCTAssertNil(error);
+    [expectation fulfill];
   }];
-  [self verifyAndRevokeToken:kAccessToken hasCallback:YES];
-  [_authorization verify];
-  [_authState verify];
-  [_tokenResponse verify];
+  [self waitForExpectationsWithTimeout:1 handler:nil];
   XCTAssertNil([_keychainHandler loadAuthState]);
 }
 
 // Verifies disconnect if access token is present.
-- (void)testDisconnectNoCallback_accessToken {
+- (void)testDisconnectNoCallback_accessTokenIsPresent {
   [_keychainHandler saveAuthState:_authState];
-  [[[_authState expect] andReturn:_tokenResponse] lastTokenResponse];
-  [[[_tokenResponse expect] andReturn:kAccessToken] accessToken];
-  [[[_authorization expect] andReturn:_fetcherService] fetcherService];
+  OCMStub([_authState lastTokenResponse]).andReturn(_tokenResponse);
+  OCMStub([_tokenResponse accessToken]).andReturn(kAccessToken);
+  
+  GIDHTTPFetcherTestBlock testBlock =
+      ^(NSURLRequest *request, GIDHTTPFetcherFakeResponseHandlerBlock responseHandler) {
+        NSData *data = [[NSData alloc] init];
+        responseHandler(data, nil);
+     };
+  [_httpFetcher setTestBlock:testBlock];
+  
   [_signIn disconnectWithCompletion:nil];
-  [self verifyAndRevokeToken:kAccessToken hasCallback:NO];
```

Comment:
Same here regarding adding the token revocation verification back.

Reply:
Same.